flink-core/src/test/java/org/apache/flink/api/common/typeutils/TypeInformationTestBase.java

```diff
@@ -21,24 +21,23 @@
 import org.apache.flink.api.common.ExecutionConfig;
 import org.apache.flink.api.common.typeinfo.TypeInformation;
 import org.apache.flink.util.InstantiationUtil;
-import org.apache.flink.util.TestLogger;
+import org.apache.flink.util.TestLoggerExtension;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 
 import java.io.IOException;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotEquals;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** Abstract test base for type information. */
-public abstract class TypeInformationTestBase<T extends TypeInformation<?>> extends TestLogger {
+@ExtendWith(TestLoggerExtension.class)
+public abstract class TypeInformationTestBase<T extends TypeInformation<?>> {
```

Comment:
This must be a separate commit, and ideally even a separate ticket.

Reply:
Yes, probably you're right
I've created a separate ticket for that https://issues.apache.org/jira/browse/FLINK-27662
and a corresponding PR https://github.com/apache/flink/pull/19744

Reply:
#19744 looks good to me; can you rebase this PR on top of the other PR? Then I can get started reviewing this one.

Reply:
yes, sure, I've  rebased it